ocean_lib/ocean/ocean.py

```diff
@@ -72,10 +72,15 @@ def __init__(self, config=None, data_provider=None):
             except AssertionError:
                 config = Config(os.getenv(ENV_CONFIG_FILE))
                 ConfigProvider.set_config(config)
-
         if isinstance(config, dict):
+            # fallback to metadataStoreUri
+            cache_key = (
```

Comment:
Can make this a one-liner: 
```
cache_key = "metadataCacheUri" if ("metadataCacheUri" in config) else "metadataStoreUri"
```

Reply:
I guess you refactored to be across 3 lines. That's ok since both the 1-line and 3-line version align with python idioms.